Task3/task1/tableController.js

```diff
@@ -0,0 +1,72 @@
+﻿const downloads = [
```

Comment:
Let's declare constant variables with upper case to simply distinguish them:
```suggestion
﻿const DOWNLOADS = [
```